src/main/app/src/Components/Navigation/Navigation.tsx

```diff
@@ -1,61 +1,70 @@
 import React from 'react';
-// import Grid from '@material-ui/core/Grid';
-import { Button, Typography, AppBar, Theme, createStyles, ButtonGroup } from '@material-ui/core';
+import { Typography, AppBar, Theme, createStyles, Box, darken } from '@material-ui/core';
 import { Link } from 'react-router-dom';
 
 import Toolbar from '@material-ui/core/Toolbar';
-import './Navigation.scss';
-
-import FavoriteIcon from '@material-ui/icons/Favorite';
 
 import makeStyles from '@material-ui/core/styles/makeStyles';
+import Search from '../Landscape/Search/Search';
+import { HelpOutlineRounded } from '@material-ui/icons';
+import IconButton from '@material-ui/core/IconButton';
+import Button from '@material-ui/core/Button';
 
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
     grow: {
       flexGrow: 1,
     },
-    title: {
-      marginRight: '20px',
+    pageTitle: {
+      padding: 11,
+      paddingLeft: 16,
+      paddingRight: 16,
+      backgroundColor: darken(theme.palette.secondary.main, 0.2),
+    },
+    menuIcon: {
       color: 'rgba(255, 255, 255, 0.75)',
     },
   })
 );
 
+interface Props {
+  appBarClass: string;
+  setSidebarContent: Function;
```

Comment:
See my comment above about the naming of the "sidebar", that in my opinion is a card showing information and not a sidebar :)

Reply:
My idea was that the sidebar could hold any information, although now mostly single cards appear. However, if you click the "log" button on the front page or the "more/extend" symbol next to the search field, other content that cards appears.

Reply:
Okay, so the sidebar contains cards. Alright ;)